### PR TITLE
A bunch of updates

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

**Comment by stdavis** [comment]
Gary tested in dev and said that everything was good. I did notice that requests to discover have a referrer value that includes the entire URL rather than just the domain. I had to update the quad word regex's to make them work.

**Comment by steveoh** [comment]
I thought the discover quad word worked on the host header or the referrer.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -8,72 +8,88 @@ on:
...
+        name: 🗝️ Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0.7.0
+        with:
+          access_token_scopes: 'email, openid, https://www.googleapis.com/auth/cloud-platform, https://www.googleapis.com/auth/firebase'
+          workload_identity_provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          create_credentials_file: true
+
+      - name: ⎔ Setup Node.js
         uses: actions/setup-node@v2
```
Comment (steveoh) [inline .github/workflows/nodejs.yml:68]:
```suggestion
        uses: actions/setup-node@v3
```

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -8,72 +8,88 @@ on:
...
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v2
+      - name: ⬇️ Checkout Code
+        uses: actions/checkout@v3
 
-      - name: Setup Node.js
+      - id: auth
+        name: 🗝️ Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0.7.0
```
Comment (steveoh) [inline .github/workflows/nodejs.yml:60]:
```suggestion
        uses: google-github-actions/auth@v0
```

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -86,40 +102,43 @@ jobs:
...
-      - name: Checkout Code
-        uses: actions/checkout@v2
+      - name: ⬇️ Checkout Code
+        uses: actions/checkout@v3
 
-      - name: Setup Node.js
-        uses: actions/setup-node@v2
+      - id: auth
+        name: 🗝️ Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0.7.0
```
Comment (steveoh) [inline .github/workflows/nodejs.yml:110]:
```suggestion
        uses: google-github-actions/auth@v0
```

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -86,40 +102,43 @@ jobs:
...
-          node-version: '14'
+          access_token_scopes: 'email, openid, https://www.googleapis.com/auth/cloud-platform, https://www.googleapis.com/auth/firebase'
+          workload_identity_provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          create_credentials_file: true
 
-      - name: Cache Node.js modules
-        uses: actions/cache@v2
+      - name: ⎔ Setup Node.js
+        uses: actions/setup-node@v2
```
Comment (steveoh) [inline .github/workflows/nodejs.yml:118]:
```suggestion
        uses: actions/setup-node@v3
```

File: src/config.js [hunk 5]
```diff
@@ -15,19 +16,21 @@ export default {
...
-    UTAH_HOUSE: 'SGID10.Political.UtahHouseDistricts2012',
-    UTAH_SENATE: 'SGID10.Political.UtahSenateDistricts2012',
-    US_CONGRESS: 'SGID10.Political.USCongressDistricts2012'
+    ZIP: 'boundaries.zip_code_areas',
+    VISTA_BALLOT_AREAS: 'political.vista_ballot_areas',
+    VISTA_BALLOT_AREAS_PROPOSED: 'political.vista_ballot_areas_proposed',
+    COUNTIES: 'boundaries.county_boundaries',
+    UTAH_HOUSE: 'political.house_districts_2022_to_2032',
+    UTAH_SENATE: 'political.senate_districts_2022_to_2032',
+    US_CONGRESS: 'political.us_congress_districts_2022_to_2032',
```
Comment (steveoh) [inline src/config.js:33]: 👌🏼👌🏼👌🏼

**Review by steveoh — APPROVED** [review]

**Pull request merged**